angler/config/bytecode-proprietary-api24.txt

```diff
@@ -2,6 +2,8 @@
 
 system/app/HwMMITest/HwMMITest.apk
 system/app/HwSarControlService/HwSarControlService.apk
+system/app/SetupSmartDeviceOverlay/SetupSmartDeviceOverlay.apk
+system/app/Tycho/Tycho.apk
```

Comment:
Added that too (https://github.com/anestisb/android-prepare-vendor/commit/cca1ca67377bad05767f522e564a49e19b8e480c) at the gplay configuration group. Also added them at bullhead configs (expect the SetupSmartDeviceOverlay which is not available there).